stable/memcached/templates/statefulset.yaml

```diff
@@ -1,5 +1,5 @@
-apiVersion: extensions/v1beta1
-kind: Deployment
+apiVersion: apps/v1beta1
+kind: StatefulSet
```

Comment:
How about we add the option to have some persistent storage given that you are moving this to a statefulset?